test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -570,7 +570,9 @@ TEST(LiteInterpreterTest, HierarchyModuleInfo) {
   // "top(C).B0(B).forward": for the add operator in B0.
   // "top(C).B0(B).A0(A).forward": for the add operator in A0.
   std::unordered_set<std::string> expected_result(
-      {"top(C).forward", "top(C).B0(B).forward", "top(C).B0(B).A0(A).forward"});
+      {"top(C).forward",
+       "top(C).B0(B).forward",
+       "top(C).B0(B).forward.A0(A).forward"});
```

Comment:
Here on the other hand it is true that A0's forward is getting called from B0's forward.